.travis.yml

```diff
@@ -10,6 +10,4 @@ go:
 
 script:
   - make build
-
-after_success:
-  - make docker-push-ci
\ No newline at end of file
+  - make docker-build
```

Comment:
Why do we create the images in travis? We don't need it

Reply:
I missunderstood and thought we wanted to build it as a test. I'll remove it.